<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gestion_emission</class>
 <widget class="QMainWindow" name="Gestion_emission">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>851</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter Emission</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>741</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="le_ID_Emission">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_Nom">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>100</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QScrollBar" name="verticalScrollBar">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>20</y>
         <width>20</width>
         <height>311</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_Ajouter">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>370</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>90</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
</string>
       </property>
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 8pt &quot;MS Shell Dlg 2&quot;;

</string>
       </property>
       <property name="text">
        <string>ID_Emission</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>66</x>
         <y>100</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;

</string>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>180</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
</string>
       </property>
       <property name="text">
        <string>Animateur</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>260</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;

</string>
       </property>
       <property name="text">
        <string>Date </string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>160</y>
         <width>221</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>180</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>260</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="eeee">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>370</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Afficher Emission</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>861</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QTableView" name="tab_Emission">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>601</width>
         <height>271</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_Ajouter_2">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>60</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>tri par nom</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_Ajouter_3">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>30</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>tri par id</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_Ajouter_4">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>90</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>tri par animateur</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_Ajouter_5">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>420</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_ID_Emission_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>420</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>420</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 8pt &quot;MS Shell Dlg 2&quot;;

</string>
       </property>
       <property name="text">
        <string>ID_Emission</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_Ajouter_6">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>120</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_stats">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>150</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Statistiques</string>
       </property>
      </widget>
      <widget class="QPushButton" name="qrcode">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>390</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>qr code</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_image_2">
       <property name="geometry">
        <rect>
         <x>686</x>
         <y>343</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="insertimg">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>430</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>inserer image</string>
       </property>
      </widget>
      <widget class="QLabel" name="imagee">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>310</y>
         <width>251</width>
         <height>161</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="update_label_linkActivated">
       <property name="geometry">
        <rect>
         <x>694</x>
         <y>180</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Arduino</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>240</y>
         <width>211</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Eliminer Emission</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>751</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID_Emission</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_ID_Emission_supp">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>80</y>
         <width>251</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>170</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>supperimer</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Modifier </string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>761</width>
        <height>431</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>721</width>
         <height>401</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLineEdit" name="lenom_mod">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>80</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="leanimateur_mod">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>180</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="ledate_mod">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>280</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_modifier">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>160</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>80</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>180</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Animateur</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>280</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Date </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID_Emission</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="leid_mod">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>20</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_recherche">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>70</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>recherche</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>quitter</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>849</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
